development/nautobot_config.py

```diff
@@ -262,6 +262,7 @@
         "barchart_bar_width": float(os.environ.get("BARCHART_BAR_WIDTH", 0.1)),
         "barchart_width": int(os.environ.get("BARCHART_WIDTH", 12)),
         "barchart_height": int(os.environ.get("BARCHART_HEIGHT", 5)),
+        "contract_types": ["Hardware Maintenance", "Subscription", "Software License", "Services", "Lease"],
```

Comment:
Propose being more clear here that there are additional contract types that are appended to the defaults `Hardware` & `Software`

```suggestion
        "additional_contract_types": ["Hardware Maintenance", "Subscription", "Software License", "Services", "Lease"],
```